statusHandler/view/termuic/termuiRenders/widgetsRender.go

```diff
@@ -273,10 +273,15 @@ func (wr *WidgetsRender) prepareListWithLogsForDisplay() {
 	wr.lLog.WrapText = true
 }
 
-func (wr *WidgetsRender) prepareLogLines(logData []string, size int) []string {
+func (wr *WidgetsRender) prepareLogLines(logData []string, maxSize int) []string {
 	logDataLen := len(logData)
-	if logDataLen > size {
-		return logData[logDataLen-size : logDataLen]
+	maxSize = maxSize - 2 // decrease 2 units as the total maxSize of the log list includes also the header and the footer
+	if maxSize <= 0 {
+		return []string{} // there isn't place for any log line
+	}
+
+	if logDataLen > maxSize {
+		return logData[(logDataLen - maxSize):logDataLen]
```

Comment:
Are you sure that this is correct? If logDataLen is higher than maxSize and you want to keep only the last maxSize characters from logData, then the correct format of this line would be: return logData[(logDataLen - maxSize):maxSize] or more simple
return logData[(logDataLen - maxSize):],  as the first parameter of this slice is the index from which we want to copy and the second one is the numbers of characters we want to copy from that index (including the character placed on that index)

Reply:
done with `logData[(logDataLen - maxSize):]`